resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -45,4 +45,6 @@
  :oidc-enable-local-admin   #boolean #or [#env LRSQL_OIDC_ENABLE_LOCAL_ADMIN false]
  :enable-clamav             #boolean #or [#env LRSQL_ENABLE_CLAMAV false]
  :clamav-host               #or [#env LRSQL_CLAMAV_HOST "localhost"]
- :clamav-port               #long #or [#env LRSQL_CLAMAV_PORT 3310]}
+ :clamav-port               #long #or [#env LRSQL_CLAMAV_PORT 3310]
+ :auth-by-cred-id           #boolean #or [#env LRSQL_AUTH_BY_CRED_ID false]
+ }
```

Comment:
Newline? (Not terribly important but still)

Reply:
removed